libc/config/linux/riscv64/entrypoints.txt

```diff
@@ -439,10 +442,6 @@ if(LLVM_LIBC_FULL_BUILD)
     libc.src.stdio.getc_unlocked
     libc.src.stdio.getchar
     libc.src.stdio.getchar_unlocked
-    libc.src.stdio.printf
```

Comment:
Does this accidentally delete `printf`?

Reply:
I think it was removed because it is listed on line 130 also.